src/msgs/handshake.rs

```diff
@@ -948,6 +948,14 @@ impl ClientHelloPayload {
         }
     }
 
+    pub fn get_params_extension(&self) -> Option<Vec<u8>> {
```

Comment:
Let's call this `get_quic_params_extension`?

Reply:
Renamed.